### PR TITLE
Update UserGuide and DeveloperGuide

### LINKED ISSUE #66: Add User Stories to DG by 29/09
Issues 1 - 3 Clifton
Issues 4 - 15 Daniel
Issues 16 - 28 Deon
Issues 29 - 34 Jason
Issues 35 - 39 Jessica (only has around 3)

Add Priority Labels to the issues (and milestones for v1.2 and v1.3)

### LINKED ISSUE #68: Add Use Cases to DG by 29/09
Add for the features/Commands assigned to you in v1.2

### PR DESCRIPTION
**Pull request by deeyonn**
Would like to merge the following changes to the master branch.

* Added edit feature to UserGuide.
* Added modify use case to the DeveloperGuide.
* Added user stories from Issues 16 to 28 to the DeveloperGuide.

Resolves #66 
Resolves #68

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -311,6 +324,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    Use case ends.
+
+**Extensions**
+
+* 1a. InternConnect detects that the specifiers given is invalid.
+
+    * 1a1. InternConnect shows an error message.
+
+      Use case ends.
+
```
Comment (lulucopter) [inline docs/DeveloperGuide.md:344]: What if the applicant doesn't exist in the database?

Comment (lulucopter) [reply docs/DeveloperGuide.md:344]: eg. INDEX is invalid

**Review by lulucopter — CHANGES_REQUESTED** [review]
Overall LGTM. Just a minor extension in the Use Case which you have to add. (can just copy from the delete use case). 

Should be able to merge once that is added!

**Review by JasonCP14 — APPROVED** [review]
LGTM

**Pull request merged**